packages/frontend/src/pages/Flows/components/CreateFlowModal.tsx

```diff
@@ -50,52 +84,58 @@ export default function CreateFlowModal(props: CreateFlowModalProps) {
       isCentered
     >
       <ModalOverlay bg="base.canvas.overlay" />
-      <ModalContent>
-        <ModalHeader p="2.5rem 2rem 1.5rem">
-          <Text textStyle="h4">Create Pipe</Text>
-        </ModalHeader>
+      <Form onSubmit={handleSubmit}>
+        <FormControl isRequired>
```

Comment:
`FormControl` is only for a single control; put this around `FormLabel` and `Input` at line 98 instead

Reply:
ok got it thanks